controllers/utils.go

```diff
@@ -56,3 +61,26 @@ func setDaemonsetNodeSelector(nodeSelector *corev1.NodeSelector, ds *appsv1.Daem
 		ds.Spec.Template.Spec.Affinity = nil
 	}
 }
+
+func getRunningPodImage(r *LVMClusterReconciler, ctx context.Context) (string, error) {
+
```

Comment:
I would recommend saving the image in the reconciler struct instead of doing this each time.

Reply:
done.